packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -7861,6 +7861,9 @@ An additional `notes` object can be passed within the `settings` object. The `no
 
 > **Note:** You can't currently manage app provisioning settings via the API. Use the administrator UI.
 
+###### Identity Store Id
```

Comment:
```suggestion
###### Identity Store ID
```

Reply:
done in https://github.com/okta/okta-developer-docs/pull/4114/commits/e00c00ad00dbc75617d9ec9517a747d2a555dd74